jib-cli/src/main/java/com/google/cloud/tools/jib/cli/jar/JarFiles.java

```diff
@@ -58,8 +58,22 @@ public static JibContainerBuilder toJibContainerBuilder(
             : Jib.from("gcr.io/distroless/java-debian10:11");
 
     List<FileEntriesLayer> layers = processor.createLayers();
-    List<String> entrypoint = processor.computeEntrypoint();
-    containerBuilder.setEntrypoint(entrypoint).setFileEntriesLayers(layers);
+    List<String> entrypoint =
```

Comment:
I think there might be some value in adding a warning when both entrypoint and jvmflags are set.

Reply:
We are currently logging a warning in `Jar#call()` but moving it to `JarFiles` seems like a good idea. It will make testing easier.

Reply:
oh, I missed that earlier, I feel like your original location is better.

Reply:
Ah I see.